questionnaire/routes.test.js

```diff
@@ -162,17 +163,35 @@ describe('Hitting /apply', () => {
     });
     describe('Instantiated questionnaire', () => {
         beforeEach(() => {
-            setUpCommonMocks();
+            setUpCommonMocks({
+                './form-helper.js': jest.fn(() => ({
+                    removeSectionIdPrefix: () => 'applicant-fatal-claim'
+                }))
+            });
         });
 
-        it('Should redirect to `/apply/resume/:questionnaireId`', async () => {
+        it('Should redirect to `/apply/resume/:questionnaireId` if there is no analytics id in session', async () => {
```

Comment:
Could you organise these tests in the same manor as the the existing one. e.g.:

````
describe('Instantiated questionnaire', () => {
    describe('Analytics ID in session', () => {
        it('Should redirect to `/apply/resume/:questionnaireId?a_id=:analyticsId`', async () => {
            ....
        });
    });
    describe('Analytics ID not in session', () => {
        it('Should redirect to `/apply/resume/:questionnaireId`', async () => {
            ....
        });
    });
});
````

this would allow you to put the `beforEach` in the relevant block and not potentially unintentionally affect other tests.